src/cpp/ext/gsm/metadata_exchange.h

```diff
@@ -40,19 +41,20 @@ class ServiceMeshLabelsInjector : public LabelsInjector {
       const opentelemetry::sdk::common::AttributeMap& map);
   // Read the incoming initial metadata to get the set of labels to be added to
   // metrics.
-  std::vector<std::pair<std::string, std::string>> GetPeerLabels(
+  std::vector<std::pair<absl::string_view, std::string>> GetPeerLabels(
       grpc_metadata_batch* incoming_initial_metadata) override;
 
   // Get the local labels to be added to metrics. To be used when the peer
   // metadata is not available, for example, for started RPCs metric.
-  std::vector<std::pair<std::string, std::string>> GetLocalLabels() override;
+  std::vector<std::pair<absl::string_view, absl::string_view>> GetLocalLabels()
+      override;
 
   // Modify the outgoing initial metadata with metadata information to be sent
   // to the peer.
   void AddLabels(grpc_metadata_batch* outgoing_initial_metadata) override;
 
  private:
-  std::vector<std::pair<std::string, std::string>> local_labels_;
+  std::vector<std::pair<absl::string_view, absl::string_view>> local_labels_;
```

Comment:
Is this actually going to be able to use `absl::string_view`?  Where is the string going to be owned?